lib/process-services/src/lib/form/services/ecm-model.service.ts

```diff
@@ -29,13 +29,15 @@ export class EcmModelService {
     public static MODEL_NAME: string = 'activitiFormsModel';
     public static TYPE_MODEL: string = 'cm:folder';
 
+    error = new EventEmitter<any>();
```

Comment:
Same here

Reply:
we don't have documentation for this service